### PR TITLE
@anyone: Updates changelogs for next release

### PR DESCRIPTION
**Pull request by ashfurrow**

### CODE REVIEW

File: docs/BETA_CHANGELOG.md [hunk 1]
```diff
@@ -1,62 +1 @@
 ## Next
-
-* Do not show the search navigation button on fullscreen artwork view. - alloy
-* Fix a crash caused by allowing the user to tap artworks in the ‘For Sale’ section of an artist that were actually
-  stale cells of the ‘Artworks’ tab. The stale cells were being shown because of assumptions about the artworks being
-  loaded before the end of the tab switch animation, which was prone to breakage on slow connections. - alloy
-* Fix a crash caused by not guarding against `nil` values in show/partner analytics data. - alloy
```
Comment (alloy) [inline docs/BETA_CHANGELOG.md]: Crash fixes can go in too imo.

File: docs/BETA_CHANGELOG.md [hunk 2]
```diff
@@ -1,62 +1 @@
...
-* Fixes a problem with view-in-room showing the back button when it was initially rotated into. - ash
-
-## 2.1.0 (2015.07.20)
-
-* Do not show the search navigation button in VIR mode. - alloy
-* Adds Adjust SDK to the app. - orta
-* Add back analytics page view event for Search and add Bell. - alloy
-* Fixes crash on expanding gene text. - ash
-* Hero units load when app connects to internet after offline start. - ash
-* Explore tab loads when app connects to the internet after offline start. - ash
```
Comment (alloy) [inline docs/BETA_CHANGELOG.md]: These two enhancements can go in too.

File: docs/BETA_CHANGELOG.md (hunk 1 above)
Comment (ashfurrow) [inline docs/BETA_CHANGELOG.md]: Cool.

File: docs/BETA_CHANGELOG.md [hunk 3]
```diff
@@ -1,62 +1 @@
...
-## 2.1.0 (2015.07.18)
-
-* Fix broken web onboarding constraints - 1aurabrown
-* Constrain size of launch logo to smaller size on iphone - 1aurabrown
-* Keep search outside of the navigation stack and maintain search results. - alloy
-* Make hit test areas of ‘MAG’, ‘YOU’, and ‘bell’ tab buttons wider. - alloy
-* Adds some spacing to the question on "do you buy art?" in onboarding - orta
-* Fix bad margins around home button - 1aurabrown
-* Add updated design for when connection errors occur. - alloy
-* Force the feed view controller to load its content when the network becomes available. - alloy
```
Comment (alloy) [inline docs/BETA_CHANGELOG.md]: This one too.

File: docs/BETA_CHANGELOG.md [hunk 4]
```diff
@@ -1,62 +1 @@
...
-* Fix top margin & whitespace gobbler for shows without install shots - 1aurabrown
-* Force portrait orientation for the admin menu on iPhone, which could be wrong on top of a landscape VIR view. - alloy
-* Don't use square placeholder image for artwork image preview. - 1aurabrown
```
Comment (alloy) [inline docs/BETA_CHANGELOG.md]: And finally this one too.

**Comment by alloy** [comment]
That owl is golden.

**Comment by ashfurrow** [comment]
It's one of my go-to happy gifs.

**Comment by ashfurrow** [comment]
Good now?

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,15 @@
 * Fix ARCollapsableTextView not expanding to full height - 1aurabrown
 * Gracefully handle cancellation of sign-in with Twitter (and presumably on device with Facebook). - ashfurrow
 * Fix personalize search bar. - 1aurabrown
+* Do not show the search navigation button on fullscreen artwork view. - alloy
+* Fixes a problem with view-in-room showing the back button when it was initially rotated into. - ash
```
Comment (alloy) [inline CHANGELOG.md]: Sorry, not these two, they were only part of the larger changes in the 2.1.0 release. I hope that clears up what entries to include?

**Comment by alloy** [comment]
Almost!!

File: CHANGELOG.md (hunk 5 above)
Comment (ashfurrow) [inline CHANGELOG.md]: No, I don't quite follow.

Comment (alloy) [inline CHANGELOG.md]: This CHANGELOG documents the app store releases, in this case 2.1.0. These two items were fixes for issues introduced _during_ the development of 2.1.0. They made sense to include in the beta version sections, but not in here as they never existed in the previous app store version.

Comment (ashfurrow) [inline CHANGELOG.md]: Ah, I see.

**Comment by alloy** [comment]
Thanks!

**Comment by alloy** [comment]
I’m merging this _after_ I’ve done the final beta build.

**Pull request merged**